spec/system/admin/enterprises_spec.rb

```diff
@@ -110,15 +110,17 @@
     accept_alert do
       click_link "Primary Details"
     end
-    uncheck 'enterprise_is_primary_producer'
+    uncheck 'enterprise_is_primary_producer' # unchecking...
```

Comment:
Interesting use of comments. It's not obvious how to connect the dots. :wink: 

What do you think about putting it together?
```suggestion
    # Unchecking hides the Properties tab.
    uncheck 'enterprise_is_primary_producer'
```

Reply:
> What do you think about putting it together?

Will do :+1: